utilities/run_unit_tests.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+#usage: ./utilities/run_unit_tests.sh [main|initial] [device|simulator]
+
+set -e
+
+echo "Invoking build script"
+echo "./utilities/build.sh "$1" "unit_tests" "$2""
```

Comment:
Add a comments if wrong parameters are given to the run_unit_tests script.
The clickup documentation mentions `./utilities/run_tests.sh [device|simulator]` while actually you need to provide 2 arguments.

Reply:
I have updated the document